src/Sulu/Bundle/SecurityBundle/Resources/js/stores/SecurityContextsStore/SecurityContextsStore.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import {Config, Requester} from 'sulu-admin-bundle/services';
+import type {SecurityContextGroups, Systems} from './types';
+
+class SecurityContextsStore {
```

Comment:
As mentioned above, `SecurityContextStore` would be more consistent with the current naming scheme.